packages/vitest/src/node/types/config.ts

```diff
@@ -573,7 +574,11 @@ export interface InlineConfig {
   /**
    * Resolve custom snapshot path
    */
-  resolveSnapshotPath?: (path: string, extension: string) => string
+  resolveSnapshotPath?: (
```

Comment:
Can we also expose a type from `vitest/config` or `vitest/node`? (most config types are exported from `vitest/node` at the moment, like coverage). Maybe `ResolveSnapshotPathHandler`?